src/containers/menu/components/routes.ts

```diff
@@ -33,110 +33,156 @@ import {
   ComponentsIconBottomNavigationDark,
 } from '@src/assets/icons';
 import { ThemeKey } from '@src/core/themes';
+import { MenuImage } from '../menuImage';
```

Comment:
refactor this to `@src/core/themes` utility
like `Theme.select`

Reply:
Done.